README.md

```diff
@@ -49,9 +49,11 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 | `grafana_tracing` | {} | [tracing](http://docs.grafana.org/installation/configuration/#tracing) configuration section |
 | `grafana_snapshots` | {} | [snapshots](http://docs.grafana.org/installation/configuration/#snapshots) configuration section |
 | `grafana_image_storage` | {} | [image storage](http://docs.grafana.org/installation/configuration/#external-image-storage) configuration section |
+| `grafana_manage_dashboards` | true | Defines if dashboards should be managed by Ansible (will overwrite exisiting) |
```

Comment:
This seems unnecessary and a bit redundant. Why not check for `grafana_dashboards | length > 0` and if there are any `"{{ grafana_dashboards_dir }}/*.json` ?